### ISSUE TITLE
Infinite $apply calls when responsive true

### ISSUE THREAD

**Issue by palsingh** [jtblin/angular-chart.js#199]
A recent change in library was to handle chart to render in a tab which is hidden. It uses $timeout to check if its parent is visible or not. When parent is visible, chart renders. It was a much needed upgrade. But now what is happening that $timeout is executing infinitely till the time all charts are not rendered and in result $timeout does is call $apply and $apply is doing its own work. It is becoming very expensive process as in my case, I've lots of filters and few tabs, So every time $timeout is done, it does $apply and $apply is executing filters and it goes on. I would suggest to add false to third param of $timeout so it should not do $apply.
Right now:
```javascript
if (isResponsive(type, scope) && container.clientHeight === 0) {
    return $timeout(function () {
        createChart(type);
    }, 50);
}
```
Should be:
```javascript
if (isResponsive(type, scope) && container.clientHeight === 0) {
    return $timeout(function () {
        createChart(type);
    }, 50, false);
}
```

I hope it make sense.

**Comment by palsingh** [comment]
I've raise a PR for it

**Comment by jtblin** [comment]
Thanks @palsingh, merged and published as `0.8.2`.
